app/src/components/custom/tables/RequestsTable.tsx

```diff
@@ -0,0 +1,61 @@
+import { 
+  Table,
+  TableBody,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Typography
+} from '@mui/material';
+import HeaderCell from './HeaderCell';
+import CustomTableCell from './CustomTableCell';
+import { ReimbursementRequest } from '../../../interfaces/ReimbursementRequest';
+import { convertStateToStatus } from '../../../utils/convertState';
+import { bcgov } from '../../../constants/colours';
+import LinkButton from '../../bcgov/LinkButton';
+import { buttonStyles } from '../../bcgov/ButtonStyles';
+
+interface RequestTableProps {
+  data: Array<ReimbursementRequest>
+}
+
+const RequestsTable = (props: RequestTableProps) => {
+  const { data } = props;
+  return (
+    <TableContainer component={Paper}>
+      <Table aria-label='simple table'>
+        <TableHead>
+          <TableRow>
+            <HeaderCell>Requestor Name</HeaderCell>
+            <HeaderCell>Items Purchased</HeaderCell>
+            <HeaderCell>Total</HeaderCell>
+            <HeaderCell>Purchase Date</HeaderCell>
+            <HeaderCell>Submission Date</HeaderCell>
+            <HeaderCell>Status</HeaderCell>
+            <HeaderCell></HeaderCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {data.length === 0 || !data
+          ? <TableRow><CustomTableCell>No requests available.</CustomTableCell></TableRow>
+          : data.map((row, index) => (
+            <TableRow
+              key={row._id}
+              sx={{ '&:last-child td, &:last-child th': { border: 0 }, backgroundColor: index % 2 === 0 ? bcgov.white : bcgov.backgroundSecondary }}
+            >
+              <CustomTableCell>{`${row.firstName} ${row.lastName}`}</CustomTableCell>
+              <CustomTableCell>{row.itemsPurchased.join('; ')}</CustomTableCell>
+              <CustomTableCell>{`$${row.totalCost}`}</CustomTableCell>
+              <CustomTableCell>{new Date(row.purchaseDate).toLocaleDateString()}</CustomTableCell>
```

Comment:
Not sure if you want this but something to look into for date formatting using functionality built into JavaScript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
Interesting option I didn't know was out there.